### PR TITLE
Update DFIRBatch.reb

### PR DESCRIPTION
**Pull request by danielgottt**
Add additional third party applications

## Description

Please include a summary of the change and (if applicable) which issue is fixed.

## Checklist:
Please replace every instance of `[ ]` with `[X]` OR click on the checkboxes after you submit your PR

- [X] I have generated a unique `GUID` for my Batch file(s)
- [X] I have tested and validated the new Batch file(s) against test data and achieved the desired output
- [X] I have placed the Batch file(s) within the `.\RECmd\BatchExamples` directory
- [X] I have set or updated the version of my Batch file(s)
- [X] I have made an attempt to document the artifacts within the Batch file(s)
- [X] I have consulted the [Guide](https://github.com/EricZimmerman/RECmd/blob/master/BatchExamples/!RECmdBatch.guide)/[Template](https://github.com/EricZimmerman/RECmd/blob/master/BatchExamples/!RECmdBatch.template) to ensure my Map(s) follow the same format

Thank you for your submission and for contributing to the DFIR community!

### CODE REVIEW

**Review by AndrewRathbun — APPROVED** [review]
Nice work, thank you!

File: BatchExamples/DFIRBatch.reb [hunk 1]
```diff
@@ -2693,6 +2693,181 @@ Keys:
...
+        KeyPath: CurrentControlSet\Services\RManService
+        Recursive: true
+        Comment: "Displays artifacts relating to RemoteUtilities"
+    -
+        Description: RemoteUtilities
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: Usoris\Remote Utilities\RManService\Host\Parameters
+        Recursive: true
+        Comment: "Displays artifacts relating to Portable RemoteUtilities Configuration"
```
Comment (reece394) [inline BatchExamples/DFIRBatch.reb:2760]: This line and the one at 2764 should be flipped

File: BatchExamples/DFIRBatch.reb [hunk 2]
```diff
@@ -2693,6 +2693,181 @@ Keys:
...
+
+# Third Party Applications -> TeamViewer - https://www.teamviewer.com/en-us/
+
+    -
+        Description: TeamViewer
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: CurrentControlSet\Services\TeamViewer
+        Recursive: true
+        Comment: "Displays artifacts relating to Splashtop"
```
Comment (reece394) [inline BatchExamples/DFIRBatch.reb:2823]: This should say TeamViewer

Comment (AndrewRathbun) [reply BatchExamples/DFIRBatch.reb:2823]: @reece394 good catch, did you want to do a PR to correct this?

File: BatchExamples/DFIRBatch.reb [hunk 3]
```diff
@@ -2693,6 +2693,181 @@ Keys:
...
+        Comment: "Displays artifacts relating to FileZilla"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> FreeFileSync - https://freefilesync.org/
+
+    -
+        Description: FreeFileSync
+        HiveType: SOFTWARE
+        Category: Third Party Applications
+        KeyPath: WOW6432Node\FileZilla Client*
```
Comment (reece394) [inline BatchExamples/DFIRBatch.reb:2862]: This should be WOW6432Node\FreeFileSync

**Comment by reece394** [comment]
Good work on this @danielgottt! But caught a few mistakes when browsing though that need corrected!

File: BatchExamples/DFIRBatch.reb (hunk 2 above)
Comment (reece394) [reply BatchExamples/DFIRBatch.reb:2823]: Currently on my phone without access to my computer so don’t want to cause issues with the file. If it still isn’t corrected tomorrow I will create a pull request fixing it

Comment (AndrewRathbun) [reply BatchExamples/DFIRBatch.reb:2823]: No worries! I can do the same later tonight too. Not a rush as the artifacts simply won't parse if they don't exist. Everything valid will still work!

File: BatchExamples/DFIRBatch.reb [hunk 4]
```diff
@@ -2693,6 +2693,181 @@ Keys:
...
+        Comment: "Displays artifacts relating to Splashtop"
+# https://jsac.jpcert.or.jp/archive/2023/pdf/JSAC2023_1_1_yamashige-nakatani-tanaka_en.pdf
+
+# Third Party Applications -> TightVNC - https://www.tightvnc.com/
+
+    -
+        Description: TightVNC
+        HiveType: SYSTEM
+        Category: Third Party Applications
+        KeyPath: tvnserver
```
Comment (reece394) [inline BatchExamples/DFIRBatch.reb:2832]: CurrentControlSet\Services\tvnserver for keypath? Don’t see tvnserver documented in that location based on the trend document

File: BatchExamples/DFIRBatch.reb (hunk 1 above)
Comment (AndrewRathbun) [reply BatchExamples/DFIRBatch.reb:2760]: Fixed here, will PR - https://github.com/AndrewRathbun/RECmd/commit/73e7eaafc80eb0d3b28cd0d4bd6bb2ad7fbf9f25

File: BatchExamples/DFIRBatch.reb (hunk 2 above)
Comment (AndrewRathbun) [reply BatchExamples/DFIRBatch.reb:2823]: https://github.com/AndrewRathbun/RECmd/commit/ab5a21d1a8603329a01e59b36cc1c82ea0b582e8

File: BatchExamples/DFIRBatch.reb (hunk 3 above)
Comment (AndrewRathbun) [reply BatchExamples/DFIRBatch.reb:2862]: https://github.com/AndrewRathbun/RECmd/commit/1974752fc4fa8b5d5be34008291a2f25f872783f

Comment (reece394) [reply BatchExamples/DFIRBatch.reb:2862]: I don’t know if the * affects RECmd output for the key or if it globs it. If it doesn’t then this needs changed to FreeFileSync without the *

**Comment by reece394** [comment]
> Good work on this @danielgottt! But caught a few mistakes when browsing though that need corrected!

These are resolved now with the latest commit

**Pull request merged**